tests/func/test_live.py

```diff
@@ -54,6 +54,11 @@ def dump(value, path):
 
 @pytest.fixture
 def live_stage(tmp_dir, scm, dvc):
```

Comment:
Need to consider not using dvclive in the dvc func tests here or creating separate integration tests for dvclive.

Reply:
Hmm, maybe we should move them into `dvclive` package